example-app/app/core/reducers/layout.ts

```diff
@@ -8,9 +8,9 @@ const initialState: State = {
   showSidenav: false,
 };
 
-export function reducer(
```

Comment:
Why is this change needed for router-store?

Reply:
Otherwise the example app will have the following error when we create the `ActionReducerMap`:
`
Type '{ layout: (state: State | undefined, action: LayoutActions) => State; router: <V extends RouterAc...' is not assignable to type 'ActionReducerMap<State, Action>'.
`